### DISCUSSION TITLE
Vulnerability checker npm integration

### DISCUSSION THREAD

**Discussion by davidFromNextLevelLab** [JetBrains/Qodana#188] [Q&A]
Hi,

is there a vulnerability checker for npm based projects?
And if it exists how do integrate it?

**Comment by brichbash** [comment]
Hello,

We bundle the Package Checker plugin with Qodana which can analyze your NPM dependencies for known vulnerabilities. To enable it, add to your `qodana.yaml` in the project root:
```
include:
  - name: VulnerableLibrariesGlobal
  - name: VulnerableLibrariesLocal
```
